hangman.py

```diff
@@ -1,213 +1,306 @@
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    # line: string
+    line = inFile.readline()
+    # wordlist: list of strings
+    wordlist = line.split()
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+# Load the list of words into the variable wordlist
+# so that it can be accessed from anywhere in the program
+wordlist = load_words()
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: list (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    # & means intersection of these 2 sets
+    if set(secret_word) == set(secret_word) & set(letters_guessed):
+        return True
+    else:
+        return False
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    word = []
+    for letter in secret_word:
+        if letter in letters_guessed:
+            word += f'{letter}'
+        else:
+            word += '_'
+    return ' '.join(word)
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    alphabet = string.ascii_lowercase
+    new_alphabet = []
+    for letter in alphabet:
+        if letter not in letters_guessed:
+            new_alphabet += letter
+    return ''.join(new_alphabet)
+
+
+def victory(guesses: int, secret_word: str):
+    """
+    guesses: count of guesses left
+    secret_word: the word the user is guessing
+    returns: string with congratulations and total score
+    """
+    style = TextStyle()
+    total_score = guesses * len(set(secret_word))
+    style.change_background_font(f'Congratulations! You won, the secret word was - {secret_word}')
+    style.change_background_font(f'Your total score is {total_score}')
+
+
+def letter_check(guesses: int, warnings: int) -> tuple:
+    """
+    guesses: count of guesses left
+    warnings: count of warnings left
+    returns: which consist of letter, updated count of guesses, updated count of warnings
+    """
+
+    # object of class text_style()
+    style = TextStyle()
+    while 1:
+        letter = input('Enter your guess letter:   ')

```

Comment:
```suggestion
        letter = input('Enter your guess letter: ')

```

Reply:
That was made to make user feel free when he typing letter. But ok